### PR TITLE
PIM-5854: The family code is not displayed at all in the product grid when no family labels

### PR DESCRIPTION
**Pull request by aRn0D**
When you create families using the csv_family_import and if you do not set any family labels, the family code is not displayed in the product grid. We should display the family code between brackets like everywhere else in the UI.

This happens only when you create families through imports. During CSV import, we import empties values instead of null values. 

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | yes
| Added Behats                      |  no
| Changelog updated                 | yes 
| Review and 2 GTM                  | yes
| Micro Demo to the PO (Story only) | no
| Migration script                  | no
| Tech Doc                          | no

### CODE REVIEW

File: src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Repository/GroupRepository.php [hunk 1]
```diff
@@ -221,7 +221,7 @@ public function createAssociationDatagridQueryBuilder()
     public function getOptions($dataLocale, $collectionId = null, $search = '', array $options = [])
     {
         $qb = $this->createQueryBuilder('o')
-            ->select('o.id as id, COALESCE(t.label, CONCAT(\'[\', o.code, \']\')) as text')
+            ->select('o.id as id, COALESCE(t.label, \'\'), CONCAT(\'[\', o.code, \']\')) as text')
             ->leftJoin('o.translations', 't', 'WITH', 't.locale=:locale')
```
Comment (pierallard) [inline src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Repository/GroupRepository.php]: You added a column here!

**Comment by juliensnz** [comment]
GTM

**Comment by jjanvier** [comment]
I don't understand 
> This happens only when you create families through imports. During CSV import, we import empties values instead of null values.

Shouldn't we fix the import too? Instead of just fixing the output in the UI?

**Comment by aRn0D** [comment]
@jjanvier yes it should be done in another PR.

**Comment by aRn0D** [comment]
TIP issue : TIP-532

**Comment by damien-carcel** [comment]
GTM

**Pull request merged**